src/mapping/nav2_map_server/src/map_reps/occ_grid_server.cpp

```diff
@@ -254,14 +254,26 @@ void OccGridServer::LoadMapFromFile(const std::string & map_name_)
   }
 
   SDL_FreeSurface(img);
+
+  rclcpp::Clock clock;
+  map_msg_.info.map_load_time = clock.now();
+  map_msg_.header.frame_id = frame_id;
+  map_msg_.header.stamp = clock.now();
+  RCLCPP_INFO(rclcpp::get_logger("map_server"), "Read a %d X %d map @ %.3lf m/cell",
+      map_msg_.info.width,
+      map_msg_.info.height,
+      map_msg_.info.resolution);  
 }
 
 void OccGridServer::ConnectROS()
 {
   // Create a publisher
-  // TODO(bpwilcox): publish a latched topic
+  rmw_qos_profile_t custom_qos_profile = rmw_qos_profile_default;
```

Comment:
I think it's worth a comment here about what you're trying to achieve with the qos settings. It might not be obvious to the reader (especially one new to ROS2) that this trying to replicate a latched topic.

Reply:
For reference: https://github.com/ros2/ros2/wiki/About-Quality-of-Service-Settings#comparison-to-ros-1

Reply:
I mean a comment in the code itself about the intent. For example, "Create a publisher using the QoS settings to emulate a ROS1 latched topic"